arch/arm/src/c5471/c5471_ethernet.c

```diff
@@ -2134,7 +2135,7 @@ static void c5471_eimreset (struct c5471_driver_s *priv)
 
   putreg32((CLKM_CTL_RST_EXT_RESET | CLKM_CTL_RST_LEAD_RESET),
            CLKM_CTL_RST);
-  up_mdelay(2);
+  up_udelay(2000);
```

Comment:
@xiaoxiang781216 do you expect better precision with `up_udelay` compared to `up_mdelay`? Or what is the motivation for this change?

Reply:
It isn't good to busy loop more than 1ms, using up_udelay with value >= 1000 will give the reader a more clean alert.